rom/dev/src/fht.rs

```diff
@@ -190,6 +191,9 @@ pub fn make_fht(env: &RomEnv) -> FirmwareHandoffTable {
         pcr_log_addr,
         // TODO: remove `pcr_bank.log_index` and just increment `fht.pcr_log_index`
```

Comment:
Can we address these TODOs in this PR as well?

Reply:
Will be easier to do that if we can get https://github.com/chipsalliance/caliptra-sw/pull/779 in first.

Reply:
Alternatively, if this goes in first, #779 will be amended to resolve this todo (already resolved the first todo).

Reply:
New TODO is gone.